tests/test_utils/tests.py

```diff
@@ -773,6 +773,10 @@ def test_count(self):
         dom2 = parse_html('<div><p>foo</p><p>bar</p></div>')
         self.assertEqual(dom2.count(dom1), 1)
 
+        # children are a superset of the tested html
+        dom2 = parse_html('<div><p>foo</p><p>bar</p><p>foo</p><p>bar</p></div>')
```

Comment:
Does it also work if you leave away the wrapping DIV? It isn't obvious to me that the issue from the ticket (`'<a/><b/>'` should be contained by `'<a/><b/><c/>'`) is being addressed by this change.

Reply:
I see the concern. Yes, with `<a/><b/><c/>` they are parsed as children of a `RootElement` which takes the place of the explicit `<div>` here, so it's the same handling. I can make that explicit by adding the test case from the ticket.

Reply:
I'm inclined to just add the test from the ticket here. 

```diff
diff --git a/tests/test_utils/tests.py b/tests/test_utils/tests.py
index 28cb64512f..7eb7302b79 100644
--- a/tests/test_utils/tests.py
+++ b/tests/test_utils/tests.py
@@ -782,6 +782,10 @@ class HTMLEqualTests(SimpleTestCase):
         dom2 = parse_html('<div><p>foo</p><p>bar</p><p>foo</p><p>bar</p></div>')
         self.assertEqual(dom2.count(dom1), 2)

+        # maybe a nice comment 
+        dom1 = parse_html('<a/><b/>')
+        dom2 = parse_html('<a/><b/><c/>')
+        self.assertEqual(dom2.count(dom1), 1)
+
     def test_parsing_errors(self):
         with self.assertRaises(AssertionError):
             self.assertHTMLEqual('<p>', '')```